webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator.go

```diff
@@ -122,7 +122,6 @@ func (v validator) ValidateCreate(ctx *context.WebhookRequestContext) admission.
 	fieldErrs = append(fieldErrs, v.validateAdvanced(ctx, vm)...)
 	fieldErrs = append(fieldErrs, v.validatePowerStateOnCreate(ctx, vm)...)
 	fieldErrs = append(fieldErrs, v.validateNextRestartTimeOnCreate(ctx, vm)...)
-	fieldErrs = append(fieldErrs, v.validateAnnotation(ctx, vm, nil)...)
```

Comment:
Isn't the goal to only drop the validation of the annotations set in the mutator?

Reply:
Hmm I thought you meant dropping the entire annotation check from creation earlier in DM (hence had the above release note). Let me update the PR to drop validating the mutated ones only.